cmd/kion/console/console.go

```diff
@@ -74,6 +76,12 @@ func run(cfg *config.Config, keyCfg *config.KeyConfig) error {
 
 	if cfg.Bool("print") {
 		fmt.Println(signinUrl)
+	} else if cfg.Bool("logout") {
+		html := fmt.Sprintf(logoutHtmlTemplate, signinUrl)
```

Comment:
Use `html/template`? Or at least use [`template.JSEscapeString`](https://pkg.go.dev/text/template#JSEscapeString) on the argument?

Reply:
Good suggested, I added `html/template`.